pkg/network/go/bininspect/symbols.go

```diff
@@ -42,7 +42,7 @@ func getSymbolNameByEntry(sectionReader io.ReaderAt, startPos, minLength int, pr
 	// If we didn't find null there, then it is not a symbol we're looking for.
 	foundNull := false
 	nullIndex := minLength
-	for ; nullIndex < len(preAllocatedBuf); nullIndex++ {
+	for ; nullIndex < readBytes; nullIndex++ {
```

Comment:
add a comment here, that `readBytes` is at most `len(preAllocatedBuf)`

Reply:
shouldn't we check the `readBytes >= nullIndex` (or minLength)?

Reply:
`readBytes > minLength` is already checked by the loop condition. If `readBytes == minLength` in the beginning of the loop, then the loop will run for zero iterations and we correctly return -1, since `minLength` does not include the NUL termination.

Reply:
Added comment.